mne/io/open.py

```diff
@@ -236,6 +236,8 @@ def _show_tree(fid, tree, indent, level, read_limit, max_str):
                 postpend = ''
                 # print tag data nicely
                 if tag.data is not None:
+                    if isinstance(tag.data, unicode):
+                        tag.data = tag.data.encode('utf-8')
                     postpend = ' = ' + str(tag.data)[:max_str]
```

Comment:
There should be a better / more uniform way to do this (Py2 and Py3 compatible) rather than doing a conditional